verilog/tools/obfuscator/verilog_obfuscate.cc

```diff
@@ -95,6 +103,20 @@ Output is written to stdout.
     return 1;
   }
 
+  // Preserve interface names (e.g. module name, port names).
+  const bool preserve_interface = absl::GetFlag(FLAGS_preserve_interface);
+  std::set<std::string> preserved;
+  if (preserve_interface) {
+    const auto status =
+        verilog::analysis::CollectInterfaceNames(content, &preserved);
+    if (!status.ok()) {
+      std::cerr << status.message();
+      return 1;
+    }
+    for (auto const& preserved_name : preserved)
+      subst.encode(preserved_name, preserved_name);
+  }
```

Comment:
minor suggestion: Enclose this block inside `{}` to help the reader quickly see that variables inside (like `preserved`) have short scope/lifetime.  Or just move `preserved` inside the `if` clause.

Reply:
You’re right! Thanks for pointing it out.